scripts/modowners/README.md

```diff
@@ -84,3 +84,10 @@ gopkg.in/square/go-jose.v2@v2.6.0
 filippo.io/age@v1.1.1
 github.com/docker/docker@v23.0.4+incompatible
 ```
+
+## Action items
+
+For existing dependencies, please review and update ownership of your team’s dependencies in `go.mod`.
+
+- If any assignments are incorrect, you can replace your team name with the correct team in `go.mod`.
+- If you don’t know who the correct team is, you can reassign the dependency back to backend platform.
```

Comment:
nit: Maybe we can suggest they open a PR to reassign to backend platform, that way it can give us a chance to review and correct it to the right team if it's not us.

Unsure if we will be blocking reviewers on any changes anyways.

Reply:
hmm maybe i'm not following - wouldn't they need to open a PR anyway to reassign back to BEP? are you saying they should add us as reviewers if they want to reassign back to us?